package.json

```diff
@@ -191,12 +186,10 @@
   "workspaces": {
     "packages": [
       "packages/**"
-    ],
-    "nohoist": [
-      "@patternfly/react-docs/react*"
     ]
   },
   "resolutions": {
-    "cssom": "0.3.4"
+    "cssom": "0.3.4",
+    "terser": "3.14.1"
```

Comment:
pr-build occasionally fails on CircleCI for no known reason. This might fix it? https://github.com/gatsbyjs/gatsby/issues/11508